tests/api-mongo.test.js

```diff
@@ -388,5 +399,30 @@ describe('MongoDB Words', function () {
           done();
         });
     });
+<<<<<<< HEAD
+});
+
+describe('API Requests For Home Directory "/"', () => {
+    describe('/undefinedRoute', () => {
+      it('should return response status of 404', (done) => {
+        chai.request('http://igboapi.com').get('/undefined').end((_, res) => {
+          expect(res.status).to.equal(404);
+          done();
+        });
+      });
+    });
+  
+    it('should return hello world!', (done) => {
+      chai.request('http://igboapi.com').get('/').end((_, res) => {
```

Comment:
Again, you should make a new command inside `tests/shared/commands.js` that will handle importing the `chai` module.

Take a look at other commands like `suggestNewWord` or `suggestNewExample`.

Reply:
Noted ...